src/blog-utils/StaticAddonCard/index.js

```diff
@@ -35,19 +45,37 @@ const StaticAddonCard = ({ addon }: Props): React.Node => {
 
       <AddonBadges addon={addon} />
 
-      <p
-        className="AddonSummary"
-        // eslint-disable-next-line react/no-danger
-        dangerouslySetInnerHTML={sanitizeHTML(nl2br(summary), ['a', 'br'])}
-      />
-
-      <GetFirefoxButton
-        addon={addon}
-        buttonType={GET_FIREFOX_BUTTON_TYPE_ADDON}
-        className="AddonFirefoxButton"
-      />
+      <div className="AddonSummary">
+        <p
+          // eslint-disable-next-line react/no-danger
+          dangerouslySetInnerHTML={sanitizeHTML(nl2br(summary), ['a', 'br'])}
+        />
+      </div>
+
+      <div className="AddonMetadata">
+        {averageRating && (
+          <Rating rating={averageRating} readOnly styleSize="small" />
+        )}
+
+        {averageDailyUsers && (
+          <p className="AddonMetadata-adu">
+            Users: {i18n.formatNumber(averageDailyUsers)}
```

Comment:
Why are we bothering to localize the format of the number when we're not even translating the string "Users:"?

Reply:
because the number must be formatted.